test/CMakeLists.txt

```diff
@@ -271,7 +271,8 @@ add_umf_test(
 # tests for the proxy library
 if(UMF_PROXY_LIB_ENABLED
    AND UMF_BUILD_SHARED_LIBRARY
-   AND NOT UMF_DISABLE_HWLOC)
+   AND NOT UMF_DISABLE_HWLOC
+   AND NOT UMF_LINK_HWLOC_STATICALLY)
```

Comment:
Why do we need to check that `NOT UMF_LINK_HWLOC_STATICALLY`?

Reply:
@vinser52 - as I recall there were some non-trivial issues with building proxy lib with static hwloc. I think we should address this later